src/watchmaker/workers/salt.py

```diff
@@ -240,8 +240,17 @@ def _get_formulas_conf(self):
                 self.salt_formula_root))[1]
         ]
 
+    def _clear_copy_dir(self, src, dst, sub):
+        source = os.sep.join((src, sub))
+        destination = os.sep.join((dst, sub))
+        if os.path.exists(source):
```

Comment:
I think if `source` doesn't exist, that's a valid error condition. We don't need to check for it, just let it raise the exception. Otherwise we get into a weird situation where it's possible for the method to "complete" without actually copying anything, and without letting the user handle the condition.

Reply:
👍